app/views/ReportView/components/TherapeuticTargets/index.scss

```diff
@@ -1,4 +1,6 @@
 .therapeutic-print {
+  margin-bottom: 50px;
```

Comment:
Looking at the margin from the patient info and tumour summary on top of this, 32px should be good instead of this seemingly arbitrary 50px

Reply:
sounds good I'll change this one